resbundler.js

```diff
@@ -107,41 +109,65 @@ function assemblFiles(dir){
     return result;
 }
 
+function parseLocale(lo){
+    if(!lo) return;
+    var list=[];
+    var locale =new Locale(lo);
+    if (locale.getLanguage()){
+        list.push(locale.getLanguage());
+    }
+    if (locale.getScript()){
+        list.push(locale.getScript());
+    }
+    if (locale.getRegion()){
+        list.push(locale.getRegion());
+    }
+    return list;
+}
+
 function dynamicFiles(){
-    if ((options.assembly != "assembled" || options.assembly == "dynamic") && options.locales.length > 0){
-        options.locales.forEach(function(lo){
-            result = "";
-            lo.split("-").map(function(v, i, a){
-                var file = a.slice(0, i+1).join("/") + "/strings.json"
-                var respath = path.join(options.resDir, file);
-                if (fs.existsSync(respath)){
-                    var data = fs.readFileSync(respath, "utf-8");
-                    var dataKey = manipulateKey(respath);
-                    if (data){
-                        var temp = dataKey +  " =  " + data + ";\n";
-                        result +=temp;
+    if ((options.assembly != "assembled" || options.assembly == "dynamic")){
+        if (options.locales.length > 0) {
+            options.locales.forEach(function(lo){
+                result = "";
+                var loList = parseLocale(lo);
```

Comment:
Take a look at https://github.com/iLib-js/iLib/blob/88b8e6701e5a9b76bb3c46474993065b3d2bbc3d/js/lib/Utils.js#L235
The locales list is a little more complicated than what you have here. Those functions will eventually be in a separate npm package that you can use here.

Reply:
You will do this in a different PR